sonar-project.properties

```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=sarvex_unpic-img
+sonar.organization=sarvex
```

Comment:
Is this your organization?

Reply:
@ascorbic yes they are from open source sonar cloud configuration. 
Need to exclude these from the PR